metrics_sdk/lib/opentelemetry/sdk/metrics/meter_provider.rb

```diff
@@ -125,6 +125,10 @@ def register_synchronous_instrument(instrument)
           end
         end
 
+        def register_asynchronous_instrument(instrument)
```

Comment:
Do we need both `register_asynchronous_instrument` and `register_synchronous_instrument`?
Could we not have just one function
```
# @api private
def register_instrument(instrument)
...
end
```

Reply:
Updated!